src/modules/init/index.ts

```diff
@@ -82,13 +93,13 @@ const promptContinue = async (repoName: string) => {
   }
 }
 
-const manifestFromPrompt = async () => {
-  return mapSeries([
+const manifestFromPrompt = async (repo: string) => {
+  return mapSeries<any, string>([
```

Comment:
What does this mean? That we're mapping functions that receive anything and return a string?

Reply:
I.......


don't know

🤣